extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -289,4 +373,20 @@ export class AmpSlideScroll extends BaseCarousel {
       }
     }
   }
+
+  /**
+   * Animate scrollLeft of the container.
+   * @param {number} fromScrollLeft.
+   * @param {number} toScrollLeft.
+   * @returns {!Promise}
+   */
+  animateScrollLeft_(fromScrollLeft, toScrollLeft) {
+    if (fromScrollLeft == toScrollLeft) {
+      return Promise.resolve();
+    }
+    const interpolate = numeric(fromScrollLeft, toScrollLeft);
+    return Animation.animate(this.slidesContainer_, pos => {
+      this.slidesContainer_./*REVIEW*/scrollLeft = interpolate(pos);
+    }, 80, 'ease-in-out').thenAlways();
```

Comment:
That's fine as long as somewhere we 100% ensure that the `scrollLeft` ends up with the right value - `toScrollLeft` in this case. This is probably already provided in the animation class - please see the api, something about halt behavior.